src/components/tooltip/tooltip.js

```diff
@@ -420,19 +424,16 @@ function MdTooltipRegistry() {
 
   /**
    * Registers a new handler with the service.
-   * @param  {String} type Type of event to be registered.
-   * @param  {Function} handler Event handler
-   * @param  {Boolean} useCapture Whether to use event capturing.
+   * @param {string} type Type of event to be registered.
+   * @param {!Function} handler Event handler.
+   * @param {boolean} useCapture Whether to use event capturing.
    */
   function register(type, handler, useCapture) {
     var array = listeners[type] = listeners[type] || [];
```

Comment:
This is a horrible variable name. Variable names should be descriptive of what it contains, not it's type. (I know if was here before, but consider changing to handlers or listenersForType.)